src/geo/leaflet/leaflet-layer-view-factory.js

```diff
@@ -5,12 +5,37 @@ var LeafletPlainLayerView = require('./leaflet-plain-layer-view');
 var LeafletCartoDBLayerGroupView = require('./leaflet-cartodb-layer-group-view');
 var LeafletTorqueLayerView = require('./leaflet-torque-layer-view');
 var LeafletCartoDBWebglLayerGroupView = require('./leaflet-cartodb-webgl-layer-group-view');
+// Should we expose a Tangram.cartodb method for doing this?
+var CCSS = require('tangram-cartocss');
 
-var LayerGroupViewConstructor = function (layerGroupModel, mapModel, options) {
-  if (options.vector) {
-    return new LeafletCartoDBWebglLayerGroupView(layerGroupModel, mapModel);
+// Make this configurable and crate a pipe of decision based on the configuration.
```

Comment:
Yes, but can a function be easily parsed? We're talking about the possibility of request that configuration from the server, if not... a regular function is perfect @javisantana